ironfish-cli/src/commands/wallet/multisig/identity/create.ts

```diff
@@ -1,18 +1,56 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
-import { ParticipantSecret } from '@ironfish/rust-nodejs'
+import { ParticipantIdentity } from '@ironfish/rust-nodejs'
+import { RPC_ERROR_CODES, RpcRequestError } from '@ironfish/sdk'
+import { CliUx, Flags } from '@oclif/core'
 import { IronfishCommand } from '../../../../command'
+import { RemoteFlags } from '../../../../flags'
 
 export class MultisigIdentityCreate extends IronfishCommand {
   static description = `Create a multisig identity`
   static hidden = true
 
-  start(): void {
-    // TODO: generate secret over RPC, persist in walletDb
-    const secret = ParticipantSecret.random()
+  static flags = {
+    ...RemoteFlags,
+    name: Flags.string({
+      char: 'n',
+      description: 'Name to assoicate with the identity',
+      required: true,
+    }),
+  }
+
+  async start(): Promise<void> {
+    const { flags } = await this.parse(MultisigIdentityCreate)
+
+    const client = await this.sdk.connectRpc()
+    let response
+    while (!response) {
+      try {
+        response = await client.wallet.multisig.createIdentity({ name: flags.name })
+      } catch (e) {
+        if (
+          e instanceof RpcRequestError &&
+          e.code === RPC_ERROR_CODES.DUPLICATE_ACCOUNT_NAME.toString()
+        ) {
+          this.log()
+          this.log(e.codeMessage)
+        }
+
+        const name = await CliUx.ux.prompt('Enter a name for the identity', {
+          required: true,
+        })
+        if (name === flags.name) {
+          this.error(`Entered the same name: '${name}'`)
+        }
```

Comment:
Nit: this is a bit redundant because, after displaying this message, it will still retry to call `createIdentity`, which will result in another error message to be printed. Maybe you meant to insert a `continue` after `this.error()`? (I don't have any strong opinion for any particular behavior)